tests/io/test_search_iterator_v3.py

```diff
@@ -0,0 +1,72 @@
+'''
+Testing the search iterators for V3 endpoints
+'''
+import pytest
+
+from tenable.io.v3.base.iterators.explore_iterator import SearchIterator
+
+ASSET_DATA = [
```

Comment:
Is that data use full?

Reply:
Thanks Alpesh for your suggestion. I have updated the test-case accordingly.